config/locales/cs.yml

```diff
@@ -0,0 +1,5446 @@
+  "false": Nepravda
+  faq: Často kladené dotazy
+  fave: přidat k oblíbeným
+  faves: Oblíbené
+  Family: čeleď
+  faved_this_observation: toto pozorování bylo přidáno k oblíbeným
+  favorite_taxa: oblíbené taxony
+  favorites: Oblíbené
+  fblocale: en_US
+  feature_this_project: Doporučit tento projekt?
+  featured: doporučený
+  featured?: doporučený?
+  featured_projects_from_everywhere: Doporučené projekty ze všech lokalit
+  featured_projects_from_place: Doporučené projekty z %{place}
+  feedback: Zpětná vazba
+  feedback_group: Skupina zpětné vazby
+  feeds: Kanály
+  feeds_for_current_filters: Kanály pro aktuální filtry
+  failed: Chyba
+  field_note: <strong>Poznámka:</strong> Pokud chcete vytvořit pole, podívejte se,
+    jestli už ho nevytvořil někdo jiný, aby nedocházelo k duplicitě.
+  fields: pole
+  fields_: Pole
+  fields_can_only_be_edited_by_the_people_who: Pole mohou upravovat pouze osoby, které
+    je vytvořily, a kurátoři stránek
+  fields_merged: Sloučená pole
+  file_should_be_in_the_following_format: 'Soubor by měl být v následujícím formátu:
+    název taxonu, popis'
+  fill_out_project_observation_fields: Vyplnit pole pozorování v projektu
+  fill_out_your_profile: vyplnit váš profil
+  filling_out_checklist_for_place: vyplnění seznamů pro místa, která dobře znáte
+  filling_out_your_profile: vyplnění vašeho profilu
+  filter: Filtrovat
+  filter_by: Filtrovat podle
+  filter_by_curator_identification: Filtrovat podle identifikace kurátora
+  filter_by_place: Filtrovat podle polohy
+  filter_by_taxon: Filtrovat podle taxonu
+  filter_by_place_taxon_and_color: Filtrovat podle místa, taxonu a barvy pomocí ovládacích
+    prvků.
+  filter_list_by_this_taxon: Filtrovat seznam podle tohoto taxonu
+  filtering: Filtrování
+  filtering_observations_that_pass_the: Filtrování pozorování, která splňují pravidla
+    projektu nebo byla provedena na místě projektu a zatím nebyla zadána
+  filters: Filtry
+  find: Najít
+  find_a_guide: Najít průvodce
+  find_a_place: Najít místo
+  find_guides: Najít průvodce
+  find_new_place_to_explore: Najít nové místo k objevování!
+  find_observations: Najít pozorování
+  find_photos: Najít fotky
+  find_places: Najít místa
+  find_some_people_to_follow: Najít někoho, koho můžete sledovat
+  find_species: Najít druhy
+  find_suitable_observations: Najít vhodná pozorování
+  find_unsuitable_observations: Najít nevhodná pozorování
+  find_your_current_location: Najít vaši aktuální lokalitu
+  finished: Dokončeno
+  finished_with_page: Práce na stránce dokončena
+  first_confirmed_inat_observation: První potvrzené pozorování %{site_name}
+  first_connect_your_provider_account_to_your_site: 'Nejdříve propojte svůj účet na
+    %{provider} se stránkou %{site_name}:'
+  first_inat_observation: První pozorování %{site_name}
+  first_observation: První pozorování
+  first_observation_by_date_added: První pozorování podle data přidání
+  fir
```

Comment:
@translatewiki these appear to be mis-translated. It should be a yaml list like we have in `en.yml`:

```yaml
    abbr_month_names:
      - 
      - Jan
      - Feb
      - Mar
      - Apr
      - May
      - Jun
      - Jul
      - Aug
      - Sep
      - Oct
      - Nov
      - Dec
```

This problems seems to be present in `cs.yml` and `el.yml`.

Reply:
Hm, I'm not sure this actually left a comment on the line that I wanted, but the problem is with `abbr_month_names` in `cs.yml` and `el.yml`.